Add triggers to prevent truncate on lookup tables

labkey-martyp opened the pull request:
#### Rationale
Truncating lookup tables can cause SND data to be orphaned without FK associations to the lookups.  These triggers throw errors in the init batch trigger.  Throwing the error is the only way to have a custom error in these batch triggers.

#### Related Pull Requests
* https://github.com/LabKey/snprcEHRModules/pull/370

#### Changes
* Add LookupSets.js with init trigger to prevent truncate
* Move Lookups.js from SNPRC to SND and add init trigger to prevent trigger.  Convert Lookups.js from EHR to standard LK trigger

labkey-martyp committed:
Add triggers to prevent truncate on lookup tables

spamhurts reviewed (approved):
Looks good. Do you know what's going on with the automated test?

labkey-martyp committed:
Adding ObjectId to Lookups test data

- Adding triggers to Lookups and LookupSets triggers results in more required column checking
